include/deal.II/matrix_free/operators.h

```diff
@@ -766,7 +766,9 @@ namespace MatrixFreeOperators
     compute_diagonal() override;
 
     /**
-     * Compute the lumped @ref GlossMassMatrix "mass matrix". This is equal to the mass matrix times a
+     * Compute the @ref GlossLumpedMassMatrix "lumped"
```

Comment:
Why not write 
```suggestion
     * Compute the @ref GlossLumpedMassMatrix "lumped mass matrix"
```
? I think a single link is less confusing and that entry links to "mass matrix" anyways.